base_technical_user/tests/test_sudo_tech.py

```diff
@@ -12,12 +12,23 @@ class SudoTechCase(SavepointCase):
     @classmethod
     def setUpClass(cls):
         super(SudoTechCase, cls).setUpClass()
+        cls.company_2 = cls.env["res.company"].create(
```

Comment:
```suggestion
        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True, no_reset_password=True))
        cls.company_2 = cls.env["res.company"].create(
```
then you can remove the other `with_context` calls